resources/release/criteria-not-met-template.md

```diff
@@ -0,0 +1,8 @@
+Hi ${RELEASE_OWNER}, </br>
+
+The below ${TYPE_OF_CRITERIA} criteria for your component has not been met. </br>
```

Comment:
can there be multiple criteria that are not being met?

Reply:
Type is either entrance or exit criteria. Starting with this. Will keep on iterating on top of this to make it list, etc.